### PR TITLE
LPD-34742 Created Provisioning Details Modal

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/ProvisioningDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+						onClick={onClose}
+						symbol="times"
+						title="Close"
+					/>
+				</span>
+			</div>
+
+			<div className="d-flex justify-content-between mb-5">
+				<OrderDetailsHeader
+					className=""
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/ProvisioningDetails.tsx:92]:
```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/ProvisioningTable.tsx [hunk 2]
```diff
@@ -299,11 +347,65 @@ const ProvisioningTable: React.FC<ProvisioningTableProps> = ({
...
+									{i18n.translate('install')}
+								</ClayButton>
+							)}
+
+						{detailsData?.isInstalled && (
+							<ClayButton
+								className="border border-danger ml-2 rounded-lg text-danger"
+								displayType="secondary"
+								onClick={() => {
+									uninstall();
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/ProvisioningTable.tsx:380]:
```suggestion
									await uninstall();
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Licenses/CreateLicense/AccountInfo/index.tsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import ClaySticker from '@clayui/sticker';
 
 import './index.scss';
 
+import ClayIcon from '@clayui/icon';
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Licenses/CreateLicense/AccountInfo/index.tsx:10]: Add it before .scss

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**